fiftyone/utils/ptv3.py

```
@@ -0,0 +1,413 @@
+            d, "point_cloud_range", default=None
+        )
+        if self.point_cloud_range is not None:
+            if len(self.point_cloud_range) != 6:
+                raise ValueError(
+                    "point_cloud_range must have 6 values "
+                    "[xmin, ymin, zmin, xmax, ymax, zmax]; found %s"
+                    % (self.point_cloud_range,)
+                )
+            lo = self.point_cloud_range[:3]
+            hi = self.point_cloud_range[3:]
+            if not all(l < h for l, h in zip(lo, hi)):
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)